mobile/src/context/BookContext.tsx

```diff
@@ -15,157 +15,18 @@ const initialState: BookState = {
-        createdAt: '1000',
-      },
-      {
-        id: 'f',
-        title: 'bar',
-        image: 'https://placekitten.com/200/305',
-        createdAt: '1000',
-      },
-      {
-        id: 'g',
-        title: 'bar',
-        image: 'https://placekitten.com/200/306',
-        createdAt: '1000',
-      },
-      {
-        id: 'h',
-        title: 'bar',
-        image: 'https://placekitten.com/200/307',
-        createdAt: '1000',
-      },
-      {
-        id: 'i',
-        title: 'bar',
-        image: 'https://placekitten.com/200/310',
-        createdAt: '1000',
-      },
-      {
-        id: 'j',
-        title: 'foo',
-        image: 'https://placekitten.com/200/300',
-        createdAt: '10000',
-      },
-      {
-        id: 'k',
-        title: 'bar',
-        image: 'https://placekitten.com/200/301',
-        createdAt: '1500',
-      },
-      {
-        id: 'l',
-        title: 'bar',
-        image: 'https://placekitten.com/200/302',
-        createdAt: '2000',
-      },
-      {
-        id: 'm',
-        title: 'bar',
-        image: 'https://placekitten.com/200/303',
-        createdAt: '3000',
-      },
-      {
-        id: 'n',
-        title: 'bar',
-        image: 'https://placekitten.com/200/304',
-        createdAt: '1000',
-      },
-      {
-        id: 'o',
-        title: 'bar',
-        image: 'https://placekitten.com/200/305',
-        createdAt: '1000',
-      },
-      {
-        id: 'p',
-        title: 'bar',
-        image: 'https://placekitten.com/200/306',
-        createdAt: '1000',
-      },
-      {
-        id: 'q',
-        title: 'bar',
-        image: 'https://placekitten.com/200/307',
-        createdAt: '1000',
-      },
-      {
-        id: 'r',
-        title: 'bar',
-        image: 'https://placekitten.com/200/310',
-        createdAt: '1000',
-      },
-      {
-        id: 's',
-        title: 'foo',
-        image: 'https://placekitten.com/200/300',
-        createdAt: '10000',
-      },
-      {
-        id: 't',
-        title: 'bar',
-        image: 'https://placekitten.com/200/301',
-        createdAt: '1500',
-      },
-      {
-        id: 'u',
-        title: 'bar',
-        image: 'https://placekitten.com/200/302',
-        createdAt: '2000',
-      },
-      {
-        id: 'v',
-        title: 'bar',
-        image: 'https://placekitten.com/200/303',
-        createdAt: '3000',
-      },
-      {
-        id: 'w',
-        title: 'bar',
-        image: 'https://placekitten.com/200/304',
-        createdAt: '1000',
-      },
-      {
-        id: 'x',
-        title: 'bar',
-        image: 'https://placekitten.com/200/305',
-        createdAt: '1000',
-      },
-
-    ];
-    dispatch({ type: 'BOOKS_LOADED', payload: books });
+    client.getBooks().then((res) => {
+      dispatch({ type: 'BOOKS_LOADED', payload: res });
+    }).catch(() => {
+      const empty: Book[] = [];
+      dispatch({ type: 'BOOKS_LOADED', payload: empty });
+    });
```

Comment:
I think in the error case we should just log a message to the console and do nothing (leave loading as `true` and don't set books to an empty array). When we set up offline mode we can have the catch statement trigger the "You are offline, functionality limited" view. No need to do that now though.

Also can you convert this to async/await?

Reply:
ah should we disable the linter for that line then? We had a console.log there but the linter complains

Reply:
ugh let's just turn that rule off for now, we can clean up the console.logs later if we come up with a better logging pipeline.

"no-console": "off" in eslintrc